### ISSUE TITLE
Possible to access isVerticalSlider() from a slider instance?

### ISSUE THREAD

**Issue by cmalven** [rcbyr/keen-slider#80]
I have a situation where I'm creating custom animation behavior using the `move` property - which I absolutely love being able to do, by the way - but this behavior requires me to know whether the slider is vertical or not so I can set the `skewX` or `skewY` transform property.

In a simple slider, I would easily know this value because I set the `vertical` option when I instantiated the slider. But what if I do the following?

```js
const slider = new KeenSlider('#my-keen-slider', {
  vertical: false,
  breakpoints: {
    '(min-width: 768px)': {
      vertical: true,
    },
    '(min-width: 1280px)': {
      vertical: false,
    },
  },
});
```

As far as I can tell, there's no way from the outside to tell whether the slider is vertical now for any given breakpoint without doing my own separate media query watching and trying to sync the values up with Keen.

Keen has an internal function called `isVertialSlider` (misspelled, BTW) but there doesn't seem to be a way to access that method, or the result of it, from outside of the slider.

Would be nice if I could do `slider.isVertical()` or `slider.details().isVertical` and get this result. If there is a way to get this value currently, and I'm missing it, please let me know.

Thanks for making a great slider. I really enjoy working with it, especially in React.

**Comment by rcbyr** [comment]
Hey @cmalven 

thank you very much for using keen-slider and reporting issues. I understand your problem and don't think you can solve this with the current implementation. So I will offer you a solution in a few days.

**Comment by rcbyr** [comment]
Hey @cmalven 

I have released a new version v5.4.0 which has a new function that provides the current options of the slider. After a breakpoint change the mounted hook is called, there you can listen to it. 

Example: https://codesandbox.io/s/cocky-volhard-ov8pg?file=/src/App.js

Hope this will solve your problem.

**Comment by cmalven** [comment]
This is great, thanks @rcbyr!
